installer/project/components/params.xml

```diff
@@ -162,6 +162,18 @@ NOTE: If you want to use the Microsoft Azure SQL Server, you must configure Tomc
         </option>
       </optionList>
     </choiceParameter>
+    <linkParameter>
+      <name>infoUpgradingTov2</name>
+      <title>Pre-installation Requirements - Upgrading from Aggregate v1.x</title>
+      <description>https://docs.opendatakit.org/aggregate-intro</description>
+      <explanation>If you are upgrading Aggregate from a v1.x version and you are using Google Account credentials, please, migrate them to regular ODK accounts before proceeding with the installation. Google Account credentials are no longer supported and can't be used to log into Aggregate v2</explanation>
+      <insertAfter>${platform}</insertAfter>
+      <clickedActionList>
+        <launchBrowser>
+          <url>http://tomcat.apache.org/download-80.cgi</url>
```

Comment:
You probably want https://docs.opendatakit.org/aggregate-intro

Reply:
Use https://docs.opendatakit.org/aggregate-upgrade/